go/test/endtoend/cluster/cluster_process.go

```diff
@@ -202,16 +209,26 @@ func (cluster *LocalProcessCluster) StartKeyspace(keyspace Keyspace, shardNames
 			log.Error(err.Error())
 			return
 		}
-
-		shards = append(shards, *shard)
+		keyspace.Shards = append(keyspace.Shards, *shard)
+	}
+	// if the keyspace is present then append the shard info
+	existingKeyspace := false
+	for idx, ks := range cluster.Keyspaces {
+		if ks.Name == keyspace.Name {
+			cluster.Keyspaces[idx].Shards = append(cluster.Keyspaces[idx].Shards, keyspace.Shards...)
+			existingKeyspace = true
+		}
+	}
```

Comment:
Since there is a req to have multiple shards in existing keyspace, I need to modify the code to add up those shards to existing shards slice.